spec/intl.html

```diff
@@ -1015,21 +1015,21 @@ <h1>ToLocalTime ( _t_, _calendar_, _timeZone_ )</h1>
             <td>[[Weekday]]</td>
             <td>
               <del>`WeekDay(tz)` specified in ES2023's <emu-xref href="#sec-week-day">Week Day</emu-xref></del>
-              <ins>! WeekDay(𝔽(truncate(_tz_ / 10<sup>6</sup>)))</ins>
```

Comment:
ToLocalTime we should leave unchanged, to still use the ECMA-262 operations, because it's not directly related to Temporal. If there are problems with NaNs here, at least they are already-existing problems in ECMA-402.